### DISCUSSION TITLE
RowList subtitle color

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#848] [Proposals]
Today is the day.

# Context
Now, rowlist subtitles are using textSecondary as a color but in cards subtitle color is textPrimary
![image](https://user-images.githubusercontent.com/6722153/195828937-90ed7a3c-1886-4119-b643-d23bea73501d.png)
![image](https://user-images.githubusercontent.com/6722153/195829005-2da4fc4d-09a3-49bf-82fb-9a00fb933b23.png)

# Problem
Inconsistency in subtitles between components (rowlist and cards)
Should we modify the rowlist subtitle color to keep consistency with the subtitle hierarchy across the different components?

# Proposal
Use textPrimary in rowList subtitles

**Comment by aweell** [comment]
![image](https://user-images.githubusercontent.com/44420072/195831628-c56d26e7-1ab1-4252-893d-daf40642dd5a.jpeg)

**Comment by AnaMontes11** [comment]
Yes, please!

**Comment by raulredondoj** [comment]
adelante, Bonaparte! además ese secondary no es AA, no?

**Comment by MontseTelefonica** [comment]
Buen cambio! Yo lo pondría rosa con parpadeo, pero vuestra opción tampoco está mal :P

**Comment by MontseTelefonica** [comment]
Un comentario a tener en cuenta:
En las snap cards, se utiliza el title y subtitle con el mismo text-present. Si ambos pasan a color textPrimary se perderá la jerarquía.
Quizás habría que cambiar el subtitle por un description.

**Reply by aweell** [reply]
Buen punto! Ahora mismo la snap ya utiliza un override a `textSecondary` para el subtitle, quizás éste si debería mantenerse...

**Reply by yceballost** [reply]
deberiamos añadir description a la snap card para cubrir los casos donde se busque usar el textSecondary

**Comment by yceballost** [comment]
Pendiente de tener screenshot test de webapp

**Reply by yceballost** [reply]
Hay bastantes usos en webapp, aunque considero que hay muchos usos incorrectos donde en realidad lo que se quiere es una description
https://devnovum.blob.core.windows.net/webapp-test-mistica-branch-failed-screenshots-1706653388858/screenshot-diffs-(iceballos-subtitle-consistency).zip
